r/R/ipc.R

```diff
@@ -107,6 +109,42 @@ read_nanoarrow.connection <- function(x, ..., lazy = FALSE) {
   check_stream_if_requested(reader, lazy)
 }
 
+#' @rdname read_nanoarrow
```

Comment:
This puts `write_nanoarrow` on `read_nanoarrow`'s page which either needs to be updated or maybe `write_nanoarrow` just needs its own help page. I'm happy to contribute the latter.